subprojects/dependency-management/src/test/groovy/org/gradle/api/internal/artifacts/configurations/DefaultConfigurationContainerSpec.groovy

```diff
@@ -180,7 +180,6 @@ class DefaultConfigurationContainerSpec extends Specification {
         then:
         detached.name == "detachedConfiguration1"
         detached.getAll() == [detached] as Set
-        detached.getHierarchy() == [detached] as Set
```

Comment:
Why remove this line?

Reply:
Must have been accidental.  I'll restore it.